ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -822,11 +822,9 @@ MiSnapThunk(IN PVOID DllBase,
             /* Now assume failure in case the forwarder doesn't exist */
             Status = STATUS_DRIVER_ENTRYPOINT_NOT_FOUND;
 
-            /* Build the forwarder name */
+            /* Build the forwarder name, including the dot */
```

Comment:
Why "including the dot"? The forwarder name is the name of the dll without the extension (i.e. without the first dot and what follows it) (Aside note: actually the extension should start at the latest dot, so I don't know whether using the strchr here is intended or not, i.e. whether windows does this or not).

Reply:
I am not sure what your point is: I am simply documenting existing code (with regard to strings), which seems consistent as far as I looked.
If you think it does not behave as it should, file an issue...